.github/workflows/pre-main.yaml

```diff
@@ -380,19 +393,15 @@ jobs:
       - name: Check out `Collector`
         uses: actions/checkout@v4
         with:
-          repository: test-network-function/collector
+          repository: shirmoran/collector
```

Comment:
Why `test-network-function/collector` is not good?

Reply:
it is temporary for testing purposes. once we'll merge the PR in the collector repo, it will be changed.